### ISSUE TITLE
0.5.0: Crash at import: Permission Denied on creation of the _d3_snapshots

### ISSUE THREAD

**Issue by gurneyalex** [mpld3/mpld3#466]
Hello

When using 0.5.0 I get a crash at import time if the user does not have write access to the installation directory. 

```
    import mpld3
  File "/usr/local/lib/python3.7/dist-packages/mpld3/__init__.py", line 50, in <module>
    os.makedirs(D3_SNAPSHOT_PATH)
  File "/usr/lib/python3.7/os.py", line 221, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/usr/local/lib/python3.7/dist-packages/_d3_snapshots/'
```

**Comment by vladh** [comment]
Hey @gurneyalex, I've just released 0.5.1 which should fix this issue. Could you please give it a try?

**Comment by shadowboxingskills** [comment]
v0.5.0 also crashed for me (same error message as @gurneyalex )
However, I installed v0.5.1 from PyPI and can confirm that the issue is fixed.
I have also tested the corresponding JS (on client side) and it works nicely as well. 
Thanks @vladh for these great updates!

**Comment by gurneyalex** [comment]
Thanks for the super fast fix @vladh, version 0.5.1 fixes the problem.
